src/appshell/view/notationstatusbarmodel.cpp

```diff
@@ -74,7 +74,12 @@ QString NotationStatusBarModel::accessibilityInfo() const
 
 QVariant NotationStatusBarModel::concertPitchItem()
 {
-    return QVariant::fromValue(makeMenuItem(TOGGLE_CONCERT_PITCH_CODE));
+    MenuItem* item = makeMenuItem(TOGGLE_CONCERT_PITCH_CODE);
+    UiActionState state;
+    state.enabled = notation() ? true : false;
+    state.checked = notation() ? notation()->style()->styleValue(StyleId::concertPitch).toBool() : false;
```

Comment:
we're actually evaluating `notation()` 3 times in 2 lines. Wouldn't we better use a local variable and do this just once?
```
$ git diff
diff --git a/src/appshell/view/notationstatusbarmodel.cpp b/src/appshell/view/notationstatusbarmodel.cpp
index 13f0460b4b..c499e16bd0 100644
--- a/src/appshell/view/notationstatusbarmodel.cpp
+++ b/src/appshell/view/notationstatusbarmodel.cpp
@@ -76,8 +76,9 @@ QVariant NotationStatusBarModel::concertPitchItem()
 {
     MenuItem* item = makeMenuItem(TOGGLE_CONCERT_PITCH_CODE);
     UiActionState state;
-    state.enabled = notation() ? true : false;
-    state.checked = notation() ? notation()->style()->styleValue(StyleId::concertPitch).toBool() : false;
+    INotationPtr ptr = notation();
+    state.enabled = ptr ? true : false;
+    state.checked = ptr ? ptr->style()->styleValue(StyleId::concertPitch).toBool() : false;
     item->setState(state);
     return QVariant::fromValue(item);
 }

```